### ISSUE TITLE
bbox mismatch in "shape" dataset

### ISSUE THREAD

**Issue by Zheweiqiu** [zylo117/Yet-Another-EfficientDet-Pytorch#159]
I randomly pick up an image(901.jpg) in validation set of "shape dataset" and draw the corresponding bounding box over the image but get an unexpected result. Below is the result。
original image
![901](https://user-images.githubusercontent.com/34045040/79823626-d3e53a80-83c6-11ea-9f3f-9d23ff59a2f8.jpg)
image with bbox
![901_new2](https://user-images.githubusercontent.com/34045040/79823658-e2cbed00-83c6-11ea-8da6-eeca3df028c8.jpg)

**Comment by zylo117** [comment]
No, you must have wrongly load the annotaions. Image id and annotation id are two different things.
